elasticsearch-stress-test.py

```diff
@@ -335,10 +356,23 @@ def main():
             # We increase the timeout here from the default value (10 seconds)
             # to ensure we wait for requests to finish even if the cluster is overwhelmed
             # and it takes a bit longer to process one bulk.
-            es = Elasticsearch(esaddress, timeout=60)
+
+            if CA_FILE:
+                context = create_ssl_context(cafile=CA_FILE)
+         
+            if AUTH_USERNAME and AUTH_PASSWORD:
+                auth = (AUTH_USERNAME, AUTH_PASSWORD)
+           
+            es = Elasticsearch(
+                esaddress,
+                http_auth=auth,
+                verify_certs=VERIFY_CERTS,
+                ssl_context=context,
```

Comment:
And this?

Reply:
And this one also line 346: context = None